Examples/php/class/runme.php

```diff
@@ -14,7 +12,7 @@
 
 # ----- Access a static member -----
 
-print "\nA total of " . Shape::nshapes() . " shapes were created\n";
+print "\nA total of " . Shape::nshapes_get() . " shapes were created\n";
```

Comment:
The `_get()` accessors are part of the "noproxy" flat wrapping, so this seems a backward change.